cmd/podman/shared/funcs.go

```diff
@@ -21,7 +22,7 @@ func GetAuthFile(authfile string) string {
 	if runtimeDir != "" {
 		return filepath.Join(runtimeDir, "containers/auth.json")
 	}
-	return ""
+	return fmt.Sprintf("/run/containers/%d/auth.json", rootless.GetRootlessUID())
```

Comment:
Is this the right directory?  At least for root I'm finding it at:
# find / -name auth.json
/run/user/0/containers/auth.json

Reply:
if root, /run/user/0/containers/auth.json, if not root /run/containers/1000/auth.json

Reply:
Can we verify this matches Podman 1.4.4 so we don't regress?

Reply:
```
# After successful 'podman login as user "tom" id=1000, non-root in another terminal

# find / -name auth.json
/run/user/1000/containers/auth.json

#  podman version
Version:            1.4.4
RemoteAPI Version:  1
Go Version:         go1.12.7
OS/Arch:            linux/amd64
```

Reply:
if `sudo -s` and login, my credentials stores in `/run/containers/0/auth.json`
```
# podman version
Version:            1.4.4
```

Reply:
Hmm.   Well OK, the plot thickens.
```
podman login (from root) :  /run/user/0/containers/auth.json
podman login (from tom) :  /run/user/1000/containers/auth.json
sudo -s podman login (from tom) : /run/containers/0/auth.json
sudo podman login (from tom) : /run/containers/0/auth.json
```
Looks like we've two possible locations for auth.json dependent on sudo?  Not sure what to do with this.

Reply:
Yes, the default path depends on whether `XDG_RUNTIME_DIR` is defined (see `getPathToAuth`), and su/sudo unsets that variable IIRC.